src/components/Snackbar/types.ts

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Colors } from '@components/types';
+
+
+export interface Props {
+
+
+    /**
+     * Any valid JSX, normally used to fire a function when clicked and dismiss snackbar
+     */
+    action?: React.ReactNode,
+
+
+    /**
+     * Custom class
+     */
+    className?: string,
+
+
+    /**
+     * Color of snackbar
+     */
+    color?: Colors,
+
+
+    /**
+     * Defaults to 0, which will not automatically dismiss
+     * 1 = 1 second, 5 = 5 seconds
+     */
+    duration?: number,
```

Comment:
would it be better for duration to take in milliseconds, like many other javascript functions do i.e. `setTimeout` and `console.time`?

Reply:
Initially, I'm annoyed with having to type milliseconds...but I think your right in keeping it consistent with the language. Will fix.